stable/drupal/requirements.lock

```diff
@@ -1,6 +1,6 @@
 dependencies:
 - name: mariadb
   repository: https://kubernetes-charts.storage.googleapis.com/
-  version: 0.7.0
-digest: sha256:e1af13ac4ac21f67582006f12d2b4eb78a1a2a59b34338fac850f2bec0b08b41
-generated: 2017-08-09T22:52:48.218901909-04:00
+  version: 1.0.5
```

Comment:
Why did the version here is 1.0.5?

Reply:
Just grabbing the latest version of mariadb which also includes the storageClass update

Reply:
Thanks, I see, just forget to update my local code..